RedHatInsights/insights-inventory-frontend
# feat(groups): groups filter edit and update for hook

The point of this PR is to make groups data available inside the entities.
That should allow us to get the same data in the consumer applications without any hard-coded changes.


How to test:
run local dev env using this branch
1)Check the functionality of the groups actions in inventory table
2)Check the functionality of the groups table
3)Check the single group page and it's functionality.

Check the redux store - the entities should have a "groups" object that is filled with data


Had to refactor the useEffect, and pull out the async function out of it. Otherwise it was breaking the groups table and causing this error https://medium.com/geekculture/react-uncaught-typeerror-destroy-is-not-a-function-192738a6e79b
